src/tests/ftest/recovery/pass_2.py

```diff
@@ -0,0 +1,211 @@
+        pcmd(hosts=dst_host, command=mkdir_cmd)
+
+        # 5-4. Update mode of the destination pool directory to 777 so that we can send
+        # the pool files.
+        self.log.info("Update mode of the fake pool directory at destination.")
+        chmod_cmd = f"sudo chmod 777 {dst_mount}/{self.pool.uuid.lower()}"
+        pcmd(hosts=dst_host, command=chmod_cmd)
+
+        # 5-5. Send the files.
+        # 1. The initial ls command lists the content of the pool directory, which
+        # contains 8 vos files (because there are 8 targets) and rdb-pool file.
+        # 2. By using xargs, each item of the ls output is passed into rsync and the rsync
+        # commands are executed in parallel. i.e., each file is sent by separate rsync
+        # process in parallel.
+
+        # - More explanations about the command:
+        # * We use --max-procs=8 to support at most 8 rsync processes to run in parallel.
+        # * -I% means replace % in the following rsync command by the output of ls. i.e.,
+        # file name.
+        # * rsync -avz means archive, verbose, and compress. By using compress, we can
+        # significantly reduce the size of the data and the transfer time.
+        # * By running rsync in parallel, we can significantly reduce the transfer time.
+        # e.g., For a 2TB pool with 8 targets per engine, each vos file size is about 7G
+        # (rdb-pool is smaller). If we run a simple rsync, which runs serially, it takes
+        # 1 min 50 sec. However, if we run them in parallel, it's reduced to 24 sec.
+        self.log.info(
+            "Copy pool files from %s:%s to %s:%s.", src_host, src_mount, dst_host,
+            dst_mount)
+        xargs_rsync_cmd = (f"ls {src_mount}/{self.pool.uuid.lower()} | "
+                           f"xargs --max-procs=8 -I% "
+                           f"rsync -avz {src_mount}/{self.pool.uuid.lower()}/% "
+                           f"{str(dst_host)}:{dst_mount}/{self.pool.uuid.lower()}")
+        pcmd(hosts=src_host, command=xargs_rsync_cmd)
+
+        # 6. Enable and start the checker.
+        dmg_command.check_enable()
+        dmg_command.check_start()
+
+        # 7. Query the checker and verify that the issue was fixed.
+        # i.e., Current status is COMPLETED.
+        errors = []
+        query_msg = ""
+        for _ in range(8):
+            check_query_out = dmg_command.check_query()
+            if check_query_out["response"]["status"] == "COMPLETED":
+                query_msg = check_query_out["response"]["reports"][0]["msg"]
+                break
+            time.sleep(5)
+        if "orphan rank" not in query_msg:
+            errors.append(
+                "Checker didn't fix orphan pool shard! msg = {}".format(query_msg))
+
+        # 8. Disable the checker.
+        dmg_command.check_disable()
+
+        # 9. Restart the servers so that the storage usage in the next step is updated.
+        dmg_command.system_start()
```

Comment:
Likewise
```suggestion
        self.server_managers[0].system_start()
```

Reply:
Actually we don't need this step, so removed it.